packages/ember-glimmer/lib/ember-template-compiler/plugins/old-syntax.js

```diff
@@ -0,0 +1,35 @@
+import { assert } from 'ember-metal/debug';
```

Comment:
From looking at the directory structure, I had just assumed that `ember-glimmer/ember-template-compiler` was the right place for anything in `packages/ember-template-compiler` that was getting ported over to glimmer2. That was the only rationale.

Pretty sure we could easily use the existing `compileOptions` instead of the new, inline one I added in `packages/ember-glimmer/lib/ember-template-compiler/system/compile.js` if that is preferable.